src/Controller/Backend.php

```diff
@@ -59,11 +59,11 @@ public function before(Request $request, Application $app)
     {
         /** @var LabelsExtension $extension */
         $extension = $app['extensions']->get('Bolt/Labels');
-        $dir = $extension->getWebDirectory()->getPath();
+        $dir = $app['paths']['root'] . $extension->getWebDirectory()->getPath();
```

Comment:
No… very specifically `$app['paths']` is a v1 approach that must not be used… that array will be removed during 3.x